src/html/htmlwin.cpp

```diff
@@ -1184,7 +1184,119 @@ void wxHtmlWindow::OnPaint(wxPaintEvent& WXUNUSED(event))
     }
 }
 
+namespace
+{
+
+// Returns true if leftCell is an ancestor of rightCell.
+bool IsAncestor(const wxHtmlCell* leftCell, const wxHtmlCell* rightCell)
+{
+    for ( const wxHtmlCell* parent = rightCell->GetParent();
+          parent; parent = parent->GetParent() )
+    {
+        if ( leftCell == parent )
+            return true;
+    }
+    return false;
+}
+
+// Returns minimum bounding rectangle of all the cells between fromCell
```

Comment:
I wonder if this method could be made simpler _and_ more efficient by returning a `wxVector<wxRect>` instead of a single one. We could just loop calling `RefreshRect()` on all of them and this would be more efficient because in the case of
```
<p>something long before "from"</p>
<p>"to" something else long after</p>
```
we could refresh just "from" and "to" instead of refreshing both paragraphs entirely (which I think happens now because even if we're careful to not take the cells before `from`, the union of 2 rectangles will still cover everything).